### PR TITLE
newdevice: Added support for Geist PDU

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #6622

### CODE REVIEW

File: includes/definitions/geist-watchdog.yaml [hunk 1]
```diff
@@ -2,9 +2,13 @@ os: geist-watchdog
...
 type: environment
 icon: geist
+group: geist
 discovery:
-    - sysObjectId:
-        - .1.3.6.1.4.1.21239.
+    -
+        sysObjectId:
+            - .1.3.6.1.4.1.21239.
+        sysDescr_except:
```
Comment (murrant) [inline includes/definitions/geist-watchdog.yaml]: Isn't this unneeded since geist-pdu is first?

Comment (laf) [inline includes/definitions/geist-watchdog.yaml]: I like to be safe, removing anyway.

Comment (laf) [inline includes/definitions/geist-watchdog.yaml]: Actually phpunit fails when I do. So leaving in place

Comment (murrant) [inline includes/definitions/geist-watchdog.yaml]: That's strange, makes it seem like I'm missing something...

Comment (murrant) [inline includes/definitions/geist-watchdog.yaml]: I removed it locally and tests ran fine.  I'm not bothered by it though.

Comment (laf) [inline includes/definitions/geist-watchdog.yaml]: I'm also now getting errors when run pre-commit about other OS' so it could be my local install :/

**Pull request merged**